bauble/src/lib.rs

```diff
@@ -0,0 +1,62 @@
+#![feature(iterator_try_collect)]
+
+pub mod convert;
+pub mod parse;
+pub mod spanned;
+pub mod value;
+
+pub use bauble_macros::FromBauble;
+
+pub use chumsky::span::SimpleSpan;
+pub use convert::{BaubleAllocator, DefaultAllocator, DeserializeError, FromBauble, VariantKind};
+pub use spanned::{SpanExt, Spanned};
+pub use value::{
+    convert_values, AssetContext, Attributes, ConvertionError, FieldsKind, Object, TypeInfo, Val,
+    Value, ValueKind,
+};
+
+use parse::Values;
+fn parse(src: &str) -> Option<Values> {
+    use ariadne::{Color, Label, Report, ReportKind, Source};
+    use chumsky::Parser;
+
+    let parser = parse::parser().then_ignore(chumsky::primitive::end());
```

Comment:
Could the `then_ignore(end())` bit here be moved into the `parser()` function itself? The only other call to it I could find is [here](https://github.com/Cakefish/bauble/blob/ae798c56c8c65baa4c41b9fdb22a7e2fb21b4a0c/bauble/src/main.rs#L19), which also has the `then_ignore(end())`.

Reply:
Yeah changed that, don't think there is any situation where you wouldn't want to do that